.github/workflows/label_stale.yml

```
@@ -0,0 +1,24 @@
+name: Label and close stale PRs and Issues
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs at midnight UTC every day
+
+jobs:
+  stale-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Seek and destroy stale PRs and Issues
+        uses: actions/stale@v4
+        with:
+          stale-pr-message: 'This pull request seems to be stale as there have been no changes in 14 days, please make changes within 7 days or the PR will be closed. If you believe this is a mistake, please inform a development team member on Discord.'
+          close-pr-message: 'This pull request was marked as stale, yet no changes have been observed in the specified time. Please feel free to open a new pull request when your project has been updated, however this pull request has been closed.'
+          stale-issue-message: 'This issue either requires verification or is unreproducible, but has had no updates for 60 days. Please provide an update within 14 days or this issue will be closed. If you believe this is a mistake, please contact an issue manager on Discord'
+          close-issue-message: 'This issue was marked as stale, yet no changes have been observed in the specified time. The issue has been closed.'
+          days-before-stale: 14
+          days-before-issue-stale: 60
+          days-before-close: 7
+          days-before-issue-close: 14
+          exempt-issue-labels: '"Stale Exempt"'
+          exempt-pr-labels: '"Stale Exempt", "-Status: Awaiting approval"'
```

Unlikely that we'd end up in a situation where these go stale, but it probably couldn't hurt to be safe
```suggestion
          exempt-pr-labels: '"Stale Exempt", "-Status: Awaiting approval", "-Status: Awaiting Merge", "-Status: Awaiting type assignment"'
```